lib/Serialization/ModuleFormat.h

```diff
@@ -636,8 +636,9 @@ enum class MacroRole : uint8_t {
   Peer,
   Conformance,
   CodeItem,
+  Extension,
 };
-using MacroRoleField = BCFixed<3>;
+using MacroRoleField = BCFixed<4>;
```

Comment:
I imagine a horrible hour of debugging behind this.

Reply:
Yes, that is EXACTLY what happened 😂